docs/metrics/README.md

```diff
@@ -356,17 +356,19 @@ predictable and reliable behavior when excessive cardinality happens, whether it
 was due to a malicious attack or developer making mistakes while writing code.
 
 OpenTelemetry has a default cardinality limit of `2000` per metric. This limit
-can be configured at `MeterProvider` level using the
-`SetMaxMetricPointsPerMetricStream` method, or at individual
-[view](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
-level using `MetricStreamConfiguration.CardinalityLimit`. Refer to this
-[doc](../../docs/metrics/customizing-the-sdk/README.md#changing-maximum-metricpoints-per-metricstream)
+can be configured at the individual metric level using the [View
```

Comment:
Should we add a note here that `MetricStreamConfiguration.CardinalityLimit` is offered as an experimental feature?. API documentation already covers that but could be useful for someone working with stable sdk package.

Reply:
either here, or the actual view doc : https://github.com/open-telemetry/opentelemetry-dotnet/pull/5328/files#r1484514169

Reply:
Added in customizing-the-sdk